parsely/src/main/java/com/parsely/parselyandroid/EventsBuilder.kt

```diff
@@ -24,15 +22,19 @@ internal class EventsBuilder(
         action: String,
         metadata: ParselyMetadata?,
         extraData: Map<String, Any>?,
-        uuid: String
+        uuid: String,
+        siteIdSource: SiteIdSource,
```

Comment:
Parameter docs for these two are missing.
Perhaps adding more details specially on how `siteIdSource` works with its default can also be helpful.